package.json

```diff
@@ -61,6 +62,7 @@
   "dependencies": {
     "@types/object-hash": "^1.3.0",
     "dataloader": "^1.4.0",
+    "lodash": "^4.14.191",
```

Comment:
Given `lodash` is kind of big, can we depend only on the camelcase package:

https://www.npmjs.com/package/lodash.camelcase